quickannotator/api/v1/annotation/utils.py

```diff
@@ -1,22 +1,63 @@
 #%%
 import threading
-import numpy as np
 import ray
 import time
 import os
 from datetime import datetime
-
+from quickannotator.db.fsmanager import fsmanager
 from quickannotator.db import get_session
 from quickannotator.db.crud.annotation import build_export_filepath
 from quickannotator.db.crud.annotation import AnnotationStore
 from quickannotator.dsa_sdk import DSAClient
-import geojson
+from quickannotator.db import db_session
 from quickannotator import constants
-from quickannotator.db.crud.image import get_image_by_id
+from quickannotator.db.crud.image import get_image_by_id, add_image_by_path
 from itertools import product
+from quickannotator.constants import IMPORT_ANNOTATION_BATCH_SIZE
 from quickannotator.db.logging import LoggingManager
+import ujson
+from quickannotator.db.crud.annotation_class import get_annotation_class_by_name_case_insensitive
+from quickannotator.db.crud.tile import TileStoreFactory, TileStore
+from tqdm import tqdm
+from shapely.geometry import shape
+
+def import_annotations(image_id: int,annotation_class_id: int, annot_file_path):
```

Comment:
Method saves 1 line of code? Is there a reason why the AnnotationStore line of code can't go inside of `import_geojson_annotation_file`?